tests/hadolint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
There's a lot of repeated code in these linters.  I think they could be replaced by even a single much simpler GHA script.

Reply:
If this is just meant to test the Dockerfiles, and I remove the only one it is in here, I think I can remove this also.